admin-ui/app/dialogs/send-push-notification.html

```diff
@@ -14,7 +14,16 @@ <h4 class="modal-title" id="myModalLabel">Send push to Application One</h4>
       </div>
     </div>
 
-
+      
```

Comment:
High wakes the device and your app up.   Normal messages are only processed when the device wakes up for regular work.